demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -803,7 +804,7 @@ def test_validate_no_missing_release_notes__missing_rn_dependent_on_api_module(s
         id_set_f = tmpdir / "id_set.json"
         id_set_f.write(json.dumps(id_set_content))
         validate_manager = ValidateManager(id_set_path=id_set_f.strpath)
-        modified_files = {api_script1.yml.rel_path}
+        modified_files = {api_script1.yml.path}
```

Comment:
why is this related to the change?

Reply:
Noticed it was not actually tested, it was returning None because file did not really exist, so i fixed this test to actually work